doc/zmq.md

```diff
@@ -109,13 +119,19 @@ No authentication or authorization is done on connecting clients; it
 is assumed that the ZeroMQ port is exposed only to trusted entities,
 using other means such as firewalling.
 
-Note that when the block chain tip changes, a reorganisation may occur
-and just the tip will be notified. It is up to the subscriber to
-retrieve the chain from the last known block to the new tip. Also note
-that no notification occurs if the tip was in the active chain - this
-is the case after calling invalidateblock RPC.
+Note that for `*block` topics when the block chain tip changes,
+a reorganisation may occur and just the tip will be notified.
+It is up to the subscriber to retrieve the chain from the last known
+block to the new tip. Also note that no notification occurs if the tip
+was in the active chain - this is the case after calling invalidateblock RPC.
+In contrast the `sequence` topic publishes all block connects and disconnects.
```

Comment:
nit proposals, feel free to take or ignore
```diff
-Note that for `*block` topics when the block chain tip changes,
-a reorganisation may occur and just the tip will be notified.
+Note that for `*block` topics, when the block chain tip changes,
+a reorganisation may occur and only the tip will be notified.
 It is up to the subscriber to retrieve the chain from the last known
-block to the new tip. Also note that no notification occurs if the tip
-was in the active chain - this is the case after calling invalidateblock RPC.
-In contrast the `sequence` topic publishes all block connects and disconnects.
+block to the new tip. Also note that no notification will occur if the tip
+was in the active chain--as would be the case after calling the invalidateblock
+RPC. In contrast, the `sequence` topic publishes all block connections and
+disconnections.
```

Reply:
took most of this